src/components/Header/index.tsx

```diff
@@ -63,7 +63,7 @@ const MenuPopover: FC = () => {
     <Popover className={styles.menuPopover}>
       {({ close }) => (
         <>
-          <Popover.Button as={'div'} className={clsx(styles.trigger, DISABLE_CGOL_MOUSE_CONTROLLER)}>
+          <Popover.Button className={clsx(styles.trigger, DISABLE_CGOL_MOUSE_CONTROLLER)}>
```

Comment:
https://github.com/Magickbase/nervos-official-website/pull/256/files#diff-b74d58497974281b6e8d776b73cf7610aca5082355bbb47d06e632f0c92cd1dfR181 should be updated too, but it's out of diff range so that suggestion cannot be created on Web UI

Reply:
changed by https://github.com/Magickbase/nervos-official-website/pull/256/commits/f6d97ed845c99495c12a40f19e08beee983f162a